api/emAnno/emAnnoService.ts

```diff
@@ -0,0 +1,83 @@
+import {http} from '../lib/http'
+import * as log4jui from '../lib/log4jui'
+import {JUILogger} from '../lib/models'
+
+const logger: JUILogger = log4jui.getLogger('em-anno-service')
+
+/**
+ * Get Annotations
+ *
+ * @param annotationsPath
+ * @returns {Promise<null>}
+ */
+export async function handleGet(annotationsPath) {
+
+    try {
+        logger.info('getting annotations', annotationsPath)
+        const response = await http.get(annotationsPath)
```

Comment:
Missing typing (what model does that http call return, what is the format of the data property that gets returned?)